src/ScrollTargetValidator.cpp

```diff
@@ -0,0 +1,76 @@
+/************************************************************************/
+/*  Xania (M)ulti(U)ser(D)ungeon server source code                     */
+/*  (C) 2021 Xania Development Team                                     */
+/*  See the header to file: merc.h for original code copyrights         */
+/************************************************************************/
+#include "ScrollTargetValidator.hpp"
+#include "ObjectIndex.hpp"
+#include "ObjectType.hpp"
+#include "Target.hpp"
+
+#include <fmt/core.h>
+
+ScrollTargetValidator::ScrollTargetValidator(const struct skill_type *skill_table)
```

Comment:
I'd drop the unnecessary `struct` here, and for new code I've started using references where I can instead of the super-prevalent `pointer but I'm going to assume it's not null` that the rest of the code base uses.

```suggestion
ScrollTargetValidator::ScrollTargetValidator(const skill_type &skill_table)
```

etc :) Seriously no biggie, this is great as is. There's a few things here that could take a `const &` too, but won't ping them all. And I'm sure I've been inconsistent in my new code too...so...a suggestion at best :)